python/librmm_cffi/wrapper.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+from numba import cuda 
+import ctypes
+
+class RMMError(Exception):
+    def __init__(self, errcode, msg):
+        self.errcode = errcode
+        super(RMMError, self).__init__(msg)
+
+
+class _librmm_wrapper(object):
```

Comment:
Class names should be `CapCase`

Reply:
This naming is copied from _libgdf_wrapper, which I did not write.